src/policy/policy.cpp

```diff
@@ -215,6 +215,7 @@ bool AreInputsStandard(const CTransaction& tx, const CCoinsViewCache& mapInputs)
 
 bool IsWitnessStandard(const CTransaction& tx, const CCoinsViewCache& mapInputs)
 {
+    bool annex_data_already_found = false;
```

Comment:
What is this for?

Reply:
I'm limiting it to one annex field per tx, mostly because BIP341/342 doesn't commit to *other* inputs' annex, which means in multiparty protocols people can malleate their input into larger sizes at will.

Maybe it's fine since I capped it to ~256 wu?

Reply:
Oh I mean the variable remains false and is never set to true, is it?

Reply:
ah. fixed.